src/components/atoms/ImageComponent/Image.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import './Image.scss';
+
+// TODO Add loading on image onLoad
+
+
+const imageOnClick = useCallback(() => onClick(), [onClick]);
```

Comment:
Bruh, are you testing that? That should be inside of the component, this function don't even exist in here, that should throw you an obvious error